temporaltest/server.go

```diff
@@ -39,10 +40,43 @@ func (ts *TestServer) fatal(err error) {
 }
 
 // Worker registers and starts a Temporal worker on the specified task queue.
+// WorkflowPanicPolicy is set to worker.FailWorkflow
 func (ts *TestServer) Worker(taskQueue string, registerFunc func(registry worker.Registry)) worker.Worker {
-	w := worker.New(ts.Client(), taskQueue, worker.Options{
-		WorkflowPanicPolicy: worker.FailWorkflow,
-	})
+	ts.defaultWorkerOptions.WorkflowPanicPolicy = worker.FailWorkflow
```

Comment:
Nit: calling this function probably shouldn't have a side effect of modifying `TestServer` fields. Instead we should set any defaults necessary during initialization.